.github/workflows/coverage.yml

```diff
@@ -1,73 +0,0 @@
-name: Test Coveralls
```

Comment:
I guess that's all for testing?
I'll wait with another round of reviews when the issue is fixed.